NuGet.config

```diff
@@ -4,6 +4,8 @@
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="MyGet" value="https://www.myget.org/F/opentelemetry/api/v3/index.json" />
+    <add key="Dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
```

Comment:
we can remove this, as we won't need any packages from non nuget.org/myget.org

Reply:
removed ..